spec/features/admin/merchant_show_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'As an Admin User' do
+  describe 'admin/merchant showpage' do
+    before :each do
+      @admin = create(:admin)
+      @merchant_1 = create(:merchant)
+      @merchant_2 = create(:merchant)
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@admin)
+    end
+    it 'shows all the same information that a merchant would see' do
+      visit admin_merchants_path
+
+      within "#merchant-#{@merchant_1.id}" do
+        click_link "#{@merchant_1.name}"
+      end
+
+      expect(current_path).to eq("/admin/merchants/#{@merchant_1.id}")
```

Comment:
What about using a path helper here?

`admin_merchant_path(@merchant_1)`

Reply:
I think this path would be `admin_merchants_path(@merchant_1)`